pages/glossary/initial-segment.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Initial Segment

```

Comment:
I don't think we capitalize any but the first word in definition titles. At least not consistently...